docs/operation/upgrade.rst

```diff
@@ -9,8 +9,8 @@ Supported Versions
 ******************
 .. note::
 
-    MetalK8 supports upgrade **strictly** from one supported
-    minor version to another. For example:
+    MetalK8s supports upgrade **strictly** from one minor version to another.
+    For example:
 
     - Upgrade from 2.0.x to 2.0.x
     - Upgrade from 2.0.x to 2.1.x
```

Comment:
Minor: Maybe it would make sense to change this especially since 2.0 and 2.1 will never be released (not sure it's needed to change it for every version but I think at least we should talk about released versions)

Reply:
Yes, why not, will change it.